app/views/choice_page.scala.html

```diff
@@ -20,13 +20,12 @@
 @import uk.gov.hmrc.play.views.html._
 @import views.components.inputs.RadioOption
 
-@this(main_template: views.html.main_template)
+@this(main_template: views.html.main_template, appConfig: AppConfig)
```

Comment:
why here ?

Reply:
We need to have list of available journeys, I stay with appConfig in this file to didn't break app. We can think how we can do it differently